CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 3.2)
 # version 3.4 is required as other do not work with C++14 and clang
```

Comment:
Quick question, why don't write 3.4 here, I've been curious about this for a long time.

Reply:
Unfortunately I'm still not skilled with CMake, so I'm taking that part as is. Feel free to update it

Reply:
> Quick question, why don't write 3.4 to the cmake minimum version, but here,
> I've been curious about this for a long time.

Could you please update to 3.4 so I can approve the merge request?

Reply:
Yes and it's been done here https://github.com/Daguerreo/NodeEditor/pull/13/commits/6533f8b7d1a05279e0c595d374dde30b8d0fc481